core/jazz_acl/components/validation.js

```diff
@@ -16,23 +16,47 @@
 
 const errorHandlerModule = require("./error-handler.js")();
 
-async function validateInput(userInput) {
+function validateBasicInput(userInput) {
 
 		if (!userInput || !userInput.method) {
 			throw (errorHandlerModule.throwInputValidationError("Invalid or missing arguments"));
     }
 
-    if (!userInput.resourcePath) {
+    if (!userInput.path) {
 			throw (errorHandlerModule.throwInputValidationError("Missing the resource path"));
 		}
 
 		if (!userInput.principalId) {
 			throw (errorHandlerModule.throwForbiddenError("You aren't authorized to access this resource"));
 		}
+}
+
+function validateGetPoliciesInput(userInput) {
+  if (!userInput.query) {
+    throw (errorHandlerModule.throwInputValidationError("Query parameters are missing"));
+  }
+
+  if (!userInput.query.serviceId) {
+    throw (errorHandlerModule.throwInputValidationError("Service Id is missing"));
+  }
+}
+
+function validatePostPoliciesInput(userInput) {
+  if (!userInput.body) {
```

Comment:
`body` is usually set by API Gateway. If it's missing, it's more of a system integration error (`InternalServerError`)